presto-hive/src/test/java/io/prestosql/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -520,15 +519,23 @@ public void testHive2VersionedFullAcidTableFails()
         deleteRecursively(tablePath, ALLOW_INSECURE);
     }
 
-    private void writeDeltaFileData(File file)
-            throws IOException
+    private boolean writeFile(File file)
     {
-        if (file.getName().equals("_orc_acid_version")) {
-            Files.write(file.toPath(), "2".getBytes(UTF_8));
+        try {
+            if (file.getName().equals("_orc_acid_version")) {
+                Files.write(file.toPath(), "2".getBytes(UTF_8));
+            }
+            else if (file.getName().startsWith("000000_")) {
+                Files.write(file.toPath(), "test".getBytes(UTF_8));
```

Comment:
I think i don't understand this. Was my comment https://github.com/prestosql/presto/pull/2989#discussion_r388134353 unsound?

I don't understand why "test" is valid content for 000000_ files but not for 000001_
Isn't most files ORC?

Reply:
The test `testFullAcidTableWithOriginalFilesFails`  started failing if we don't write anything on original file (000000_). You are correct, this isn't in orc format. I can look into adding ORC content.